src/validations/rules/ssp.sch

```diff
@@ -1661,6 +1608,46 @@
                 test="current() = $security-objective-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security objective
                 value.</sch:assert>
         </sch:rule>
+        <sch:rule
+            context="o:control-implementation/o:implemented-requirement/o:statement/o:by-component/o:set-parameter"
```

Comment:
`set-parameter` elements are children of `implemented-requirement` and its descendant `by-component` elements.

Your context incorrectly states they are strictly subordinate to `by-component`.

Correct the context.

The correct context is `o:control-implementation/o:implemented-requirement/descendant-or-self::o:set-parameter`.

At a subsequent time - and a separate issue - `set-parameter` elements subordinate to `control-implementation` will need attention.